hypothesis_1.ipynb

```diff
@@ -3346,6 +3362,19 @@
     "group the results to analyze the first.\n",
     "'''\n",
     "\n",
+    "# AW: I don't think we can make a direct mapping between ORES articletopic\n",
```

Comment:
Yes. After our discussion, I updated this and the new results test against the ORES tool and it also use the topics taxonomy exclusively. Thanks.